src/pages/home/report/ReportActionsView.js

```diff
@@ -252,15 +319,39 @@ class ReportActionsView extends React.Component {
         }
 
         this.updateSortedReportActions();
+        const floatingReportActionContextMenuWrapperStyle = getFloatingReportActionContextMenuWrapperStyles(
+            this.state.reportActionContextMenuVerticalPosition,
+        );
         return (
-            <InvertedFlatList
-                ref={el => this.actionListElement = el}
-                data={this.sortedReportActions}
-                renderItem={this.renderItem}
-                contentContainerStyle={[styles.chatContentScrollView]}
-                keyExtractor={item => `${item.action.sequenceNumber}`}
-                initialRowHeight={32}
-            />
+            <>
+                <InvertedFlatList
+                    ref={el => this.actionListElement = el}
+                    data={this.sortedReportActions}
+                    renderItem={this.renderItem}
+                    onScroll={this.updateReportActionContextMenuPosition}
+                    scrollEventThrottle={32}
```

Comment:
Another implementation note, because this `scrollEventThrottle` is actually pretty important...

When I first implemented this w/ the default value of `0` for `scrollEventThrottle`, there was a noticeable performance degradation of the `ScrollView`, where scrolling was just not smooth at all and it was skipping a lot of frames. The reason for that performance degradation is because I was running `measureInLayout` on every frame, which I think is a slightly expensive asynchronous function.

So this prop throttles the scroll event so that the `onScroll` callback isn't called as frequently. If you give it a lower value, the `onScroll` callback is called more frequently, and the visible result is that scrolling is less smooth but the floating ReportActionContextMenu is more responsive/snappy. If you give it a higher value, the `onScroll` callback is called less frequently, and the visible result is that scrolling is smoother, but the floating ReportActionContextMenu is sluggish and slower to adjust its position.

I experimented with this value a bit, and 32 felt like a good balance where scrolling was still smooth and the `ReportActionContextMenu` wasn't noticeably sluggish.

Reply:
I can get down with that approach.